user_guide_src/source/installation/running.rst

```diff
@@ -83,156 +86,200 @@ The local development server can be customized with three command line options:
+
+Hosting with mod_userdir (Shared Hosts)
+=======================================
 
 A common practice in shared hosting environments is to use the Apache module "mod_userdir" to enable per-user Virtual Hosts automatically. Additional configuration is required to allow CodeIgniter4 to be run from these per-user directories.
 
 The following assumes that the server is already configured for mod_userdir. A guide to enabling this module is available `in the Apache documentation <https://httpd.apache.org/docs/2.4/howto/public_html.html>`_.
 
 Because CodeIgniter4 expects the server to find the framework front controller at **public/index.php** by default, you must specify this location as an alternative to search for the request (even if CodeIgniter4 is installed within the per-user web directory).
 
-The default user web directory **~/public_html** is specified by the ``UserDir`` directive, typically in **apache2/mods-available/userdir.conf** or **apache2/conf/extra/httpd-userdir.conf**::
+The default user web directory **~/public_html** is specified by the ``UserDir`` directive, typically in **apache2/mods-available/userdir.conf** or **apache2/conf/extra/httpd-userdir.conf**:
+
+.. code-block:: apache
 
     UserDir public_html
 
-So you will need to configure Apache to look for CodeIgniter's public directory first before trying to serve the default::
+So you will need to configure Apache to look for CodeIgniter's public directory first before trying to serve the default:
+
+.. code-block:: apache
 
     UserDir "public_html/public" "public_html"
 
-Be sure to specify options and permissions for the CodeIgniter public directory as well. A **userdir.conf** might look like::
+Be sure to specify options and permissions for the CodeIgniter public directory as well. A **userdir.conf** might look like:
+
+.. code-block:: apache
 
     <IfModule mod_userdir.c>
         UserDir "public_html/public" "public_html"
         UserDir disabled root
 
         <Directory /home/*/public_html>
-                AllowOverride All
-                Options MultiViews Indexes FollowSymLinks
-                <Limit GET POST OPTIONS>
-                        # Apache <= 2.2:
-                        # Order allow,deny
-                        # Allow from all
-
-                        # Apache >= 2.4:
-                        Require all granted
-                </Limit>
-                <LimitExcept GET POST OPTIONS>
-                        # Apache <= 2.2:
-                        # Order deny,allow
-                        # Deny from all
-
-                        # Apache >= 2.4:
-                        Require all denied
-                </LimitExcept>
+            AllowOverride All
+            Options MultiViews Indexes FollowSymLinks
+            <Limit GET POST OPTIONS>
+                # Apache <= 2.2:
+                # Order allow,deny
+                # Allow from all
+
+                # Apache >= 2.4:
+                Require all granted
```

Comment:
Ah I see, it looks like this is a new approach. I trust you on this.

Reply:
What do you mean? The change here is simply a change in indentation.